src/components/molecules/RadioField/__snapshots__/RadioField.test.tsx.snap

```diff
@@ -146,6 +129,24 @@ exports[`<RadioField /> renders the component with no control icon and no a11y v
   animation: ciAMHu 200ms ease-in-out;
 }
 
+.c2:hover:not(:disabled) + label,
+.c2:focus + label {
+  border-color: #009E8E;
+  box-shadow: 0 0 4px 0 #009E8E;
+  background-color: transparent;
```

Comment:
This property didn't exist in the Zopa theme, we comfortable with the scope of this change?